pandas/core/algorithms.py

```diff
@@ -170,6 +170,11 @@ def _ensure_data(values: ArrayLike) -> np.ndarray:
         return np.asarray(values)
 
     elif is_complex_dtype(values.dtype):
+        # error: Item "ExtensionDtype" of "Union[Any, ExtensionDtype]"
+        # has no attribute "itemsize"
+        if values.dtype.itemsize == 16:  # type: ignore[union-attr]
+            # We have support for complex128
```

Comment:
not complex64?

Reply:
I'd be happy to add complex64 to the test suite, but thought it better to do one case (complex128) at a time.  I didn't want to add something (`values.dtype.itemsize == 8`) that wasn't tested by the test suite.

Reply:
not a deal-breaker, but id prefer to do them both in this pass.  or at least leave a comment explaining why one is excluded